3_etl_code/ETL_Orchestration/sql/etl_measurement.sql

```diff
@@ -99,16 +105,25 @@ SELECT
 # visit_detail_id
   NULL AS visit_detail_id,
 # measurement_source_value
-  mfrwsasci.CODE1 AS measurement_source_value,
+ CASE
+      WHEN mfrwsasci.SOURCE IN ("BMI","HEIGHT") THEN mfrwsasci.code
+      ELSE mfrwsasci.CODE1
+  END AS measurement_source_value,
 # measurement_source_concept_id
   CASE
     WHEN mfrwsasci.omop_source_concept_id IS NOT NULL THEN CAST(mfrwsasci.omop_source_concept_id AS INT64)
     ELSE 0
   END AS measurement_source_concept_id,
 # unit_source_value
-  CAST(NULL AS STRING) AS unit_source_value,
+  CASE
+       WHEN mfrwsasci.SOURCE = "HEIGHT" THEN "cm"
+       ELSE CAST(NULL AS STRING)
+  END AS unit_source_value,
 # unit_source_concept_id
-  0 AS unit_source_concept_id,
+  CASE
+       WHEN mfrwsasci.SOURCE = "HEIGHT" THEN 2001800125
```

Comment:
dotn like the idea of hardcoding a concept_id code in the etl code

Reply:
Yes, this one has to be fixed within in the `stem_medical_events` table as this is new.